test.js

```diff
@@ -1,19 +1,21 @@
-var project = 'project-require'
-  , require = require(project)
-  ;
+var project = 'project-require',
+    projectFile = './index.js'
+    , require = require(projectFile);
 
 // check default behavior
 console.assert(require('/package.json').name == project);
 
 // check custom path
-require.projectPath(__dirname+'/node_modules/'+project);
-console.assert(require('/../../package.json').name == project);
+require.projectPath(__dirname + '/index.js');
+console.assert(require('/../package.json').name == project);
```

Comment:
Idea here was to emulate using it from within a module of an app, but this this change it lost the purpose.